base/applications/regedit/framewnd.c

```diff
@@ -1368,26 +1363,24 @@ static BOOL _CmdWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
         RegKeyEditPermissions(hWnd, hKeyRoot, NULL, keyPath);
         break;
     case ID_REGISTRY_PRINTERSETUP:
-        /*PRINTDLG pd;*/
-        /*PrintDlg(&pd);*/
-        /*PAGESETUPDLG psd;*/
-        /*PageSetupDlg(&psd);*/
+        //PRINTDLG pd;
+        //PrintDlg(&pd);
+        //PAGESETUPDLG psd;
+        //PageSetupDlg(&psd);
         break;
     case ID_REGISTRY_OPENLOCAL:
         break;
-
     case ID_VIEW_REFRESH:
         RefreshTreeView(g_pChildWnd->hTreeWnd);
         keyPath = GetItemPath(g_pChildWnd->hTreeWnd, 0, &hKeyRoot);
         RefreshListView(g_pChildWnd->hListWnd, hKeyRoot, keyPath, TRUE);
         break;
-        /*case ID_OPTIONS_TOOLBAR:*/
-        /*	toggle_child(hWnd, LOWORD(wParam), hToolBar);*/
-        /*    break;*/
+        //case ID_OPTIONS_TOOLBAR:
+        //   toggle_child(hWnd, LOWORD(wParam), hToolBar);
+        //   break;
```

Comment:
```suggestion
        //    toggle_child(hWnd, LOWORD(wParam), hToolBar);
        //    break;
```
(indent is 4 spaces, check how stuff is aligned below "case XXX:" above)